src/traveler.js

```diff
@@ -17,9 +17,10 @@ class Traveler {
   determineTrips(trips, currentDate, allDestinations) {
     trips.forEach(trip => {
       let tripDestination
-      allDestinations.forEach(destination => {
```

Comment:
this probably works better as a `forEach( )` since you are not returning anything, but just my opinion!